README.markdown

```diff
@@ -34,9 +34,16 @@ Please contact Wenbin Yang, bysin7 (at) gmail (dot) com with questions or feedba
 
 Step by step install With Visual Studio 2017 for Windows 7/10 64 Bit
 -----------------------------------------------------------------
-* Install OpenCV 3.0 or later
-* Install OpenGL with GLFW3.0; (Check [this great website](https://learnopengl.com/) for more details.)
+* Install OpenCV 3.0 or later;Click [here](https://opencv.org/) and say hello to OpenCV
+  * Check this [link](https://docs.opencv.org/) and find what you need. There are many great tutorials in this website.
+  * If you configure all libs and includes with OpenCV successfully, installation of OpenGL and Pylon 5 will be very easy for you.
+    Other two processes are similar to this one.
+* Install OpenGL with GLFW3.0 or later
+  * Follow the tutorial in [this website](https://learnopengl.com/). Do not forget to install GLAD after installation of GLFW. It won't     cost you too much time.
+  * tips: You can download Windows pre-compiled binaries as an alternative. Click "64-bit Windows binaries" to download it. 
```

Comment:
I've tried the pre-compiled binaries, however, it didn't work in our case. Have you tried?

Reply:
Two sample  code in the learningopengl website run correctly. But I don't know whether it would work in BLITZ,  should I try it this afternoon?

Reply:
Not necessary, what we should do is to provide a valid solution for them. So it doesn't have to be multiple options for them.

Reply:
Okay, but if it doesn't work in your case, I don't know whether my configuration is correct or not. Maybe I should try it some time in the future.